### PR TITLE
modernize/correct recipes with cutting board & fix tool integration issue

### PR DESCRIPTION
**Pull request by Shibva**
this has bugged me for the longest time so I decided to put the time in and make a needed fix that I think you will agree with

This PR will do the following
- Reformated all recipes to use the tag without `toos/` in it (ie: #c:tools/axes --> #c:axes)
  - corrects an issue with other mods tools not working as they would use `#c: directly` instead of `#c:tools/`
- migrated `#c:tools/` contents into `#c` directory (so things can work still even if no other mods exist to expand the list) 
- removed `#c:tools/` as it is now redundant
- update `tools.json` to account for the change of location of its targeted contents (ensuring everything is still together)

TLDR: Just takes everything and steps its tags back a notch. 

Tested and is working as it should; this will 
- Ensure other tools are usable as `#c:tools/` is not or is no longer used as the primary tool directory
- changes should be applicable directly in 1.20.x branches

### CODE REVIEW

**Comment by Shibva** [comment]
To further elaborate; in this version, none of them mods that tag their items used `c:tools/` and therefore not allowing for proper compatibility with other moded tools that would be acceptable.

These changes here should fix that for not only the recipes but including updated tags themselves in case any other mod that would add them, naturally, is not present.
